src/meta/dashboard/Cargo.toml

```diff
@@ -14,7 +14,7 @@ axum = { workspace = true }
 axum-embed = "0.1"
 bytes = "1"
 mime_guess = "2"
-reqwest = "0.12.2"
+reqwest = "0.12"
```

Comment:
Why do we need to allow 0.12.0 / 0.12.1 here?

Reply:
A leftover from the previous version, I rolled back this change